src/main/java/ch/astorm/jotlmsg/io/FlatEntryListStructure.java

```diff
@@ -0,0 +1,147 @@
+package ch.astorm.jotlmsg.io;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Represents an [MS-OXCDATA] 2.3.3 FlatEntryList Structure in Java.
+ * 
+ * @author Guido Stein
+ *
+ */
+public class FlatEntryListStructure<T extends FlatEntryStructure> implements Iterable<T> {
+    private long count; // Number of FlatEntryStructures.
+    private long size; // Number of bytes of all FlatEntryStructures.
+    private List<T> flatEntryStructures = new ArrayList<T>();
+    private FlatEntryStructureFactory<T> fesf = new FlatEntryStructureFactory<T>();
```

Comment:
Use the diamond operator => `new ArrayList<>()`.

Reply:
OK.